root/io/issue-7754/read_cmssw_class_v3.cxx

```diff
@@ -59,10 +59,10 @@ void Print(const Holder &h)
 }
 
 #ifdef __ROOTCLING__
-#pragma read sourceClass="SubContent" targetClass="SubContent" versions="[2]" source="int fValue" target="fValue" code="{ fValue = 100 * onfile.fValue; }"
```

Comment:
The changes of this commit do not seem to be related to the title of the PR, could you elaborate?

Reply:
I can't remember the exact circunstances (too long ago) but this issues in the test where discovered as part of this investigation (i.e indeed they could have been a separate PR).